bucket/gittyup.json

```
@@ -0,0 +1,36 @@
+{
+    "version": "1.1.0",
+    "description": "Gittyup is a graphical Git client designed to help you understand and manage your source code history. Gittyup is a continuation of the GitAhead client.",
+    "homepage": "https://murmele.github.io/Gittyup/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Murmele/Gittyup/releases/download/stable/Gittyup-win64-1.1.0.exe#/dl.7z",
+            "hash": "596270fa68ebff7e5a920d69800693b32ae033fd32f496f3f16b0be95de016aa"
+        },
+        "32bit": {
+            "url": "https://github.com/Murmele/Gittyup/releases/download/stable/Gittyup-win32-1.1.0.exe#/dl.7z",
+            "hash": "571ccd71ba5881b81206ec9e03d6a99d6b6d21850963976b1a34ac1f9f1fa359"
+        }
+    },
+    "checkver": {
```

```suggestion
    "checkver": {
        "url": "https://api.github.com/repos/Murmele/Gittyup/releases",
        "regex": "Gittyup-win64-([\\d.]+)\\.exe"
```
Scoop matches `tag/v?([\\d.]+)` in the **latest release** by default. So using "github" would have some problems:
(1) The [latest release](https://github.com/Murmele/Gittyup/releases/) of *Gittyup* is the dev version, which is not what we want.
(2) Their tag name is simply "stable", so we have to match the version either from "title" or from "file name".
I use the API page instead of `https://github.com/Murmele/Gittyup/releases/` because it is more compact.